src/plugins/vis_types/timeseries/public/application/components/yes_no.tsx

```diff
@@ -23,7 +23,7 @@ export function YesNo({
   name,
   value,
   disabled,
-  'data-test-subj': dataTestSubj,
+  'data-test-subj': dataTestSubj = name,
```

Comment:
Had to assign a default `data-test-subj` value to avoid having `undefined-yes` or `undefined-no` for all YesNo component on functional testing.
It should not affect existing tests as it only kicks in when the test id is not passed.